src/ingest_validation_tools/validation_utils.py

```diff
@@ -176,7 +173,10 @@ def get_data_dir_errors(
     if {x.name for x in shared_directories} == expected_shared_directories:
         # If they exist create a list of the paths
         data_paths = list(shared_directories)
-    # Otherwise, do nothing we can just use the predefine data_path
+    # Otherwise, do nothing we can just use the predefined data_path
+
+    for data_path in data_paths:
+        assert Path(data_path).exists, f"data_path {data_path} does not exist!"
```

Comment:
same question RE assertion

Reply:
This one is caught where `get_data_dir_errors` is called in upload.py (inside try block in _check_data_path).

Reply:
Actually this didn't make the cut either, removed.